processing/src/test/java/org/apache/druid/query/aggregation/FilteredAggregatorTest.java

```diff
@@ -84,6 +85,40 @@ public void testAggregate()
     assertValues(agg, selector, expectedFirst, expectedSecond, expectedThird);
   }
 
+  @Test
+  public void testAggregateWithNullableSelectorAndNullValues()
+  {
+    NullHandling.initializeForTestsWithValues(false, true);
+
+    final TestFloatNullableColumnSelector selector = new TestFloatNullableColumnSelector(null);
+
+    final FilteredAggregatorFactory factory = new FilteredAggregatorFactory(
+        new FloatMaxAggregatorFactory("loss", "value"),
+        new SelectorDimFilter("dim", "a", null)
+    );
+
+    final FilteredAggregator agg = (FilteredAggregator) factory.factorize(makeColumnSelector(selector));
+    aggregate(selector, agg);
+    assertValues(agg, selector);
+  }
+
+  @Test
+  public void testAggregateWithNullableSelectorAndNonNullValues()
```

Comment:
nit: since isNull on the selector always returns true is this really testing null and non-null values?